sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Set-SPOFontPackage.md

```diff
@@ -0,0 +1,109 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spofontpackage
+applicable: SharePoint Online
+title: Set-SPOFontPackage
+schema: 2.0.0
+author: luchaoqiu
+ms.author: luchaoqiu
+ms.reviewer:
+---
+
+# Set-SPOFontPackage
+
+## SYNOPSIS
+
+Applies a brand font package to a SharePoint site or Viva Connections.
+
+## SYNTAX
+
+```
+Set-SPOFontPackage [-Identity] <SPOFontPackagePipeBind> [-WebUrl] <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Set-SPOFontPackage` cmdlet applies a brand font package to a specified SharePoint site or Viva Connections. Use this cmdlet to customize the typography and branding of SharePoint sites by applying predefined font packages.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Set-SPOFontPackage -Identity 12345678-1234-1234-1234-123456789012 -WebUrl "https://contoso.sharepoint.com/sites/marketing"
+```
+
+This example applies the font package with the specified GUID to the marketing site.
+
+### EXAMPLE 2
+
+```powershell
+$sites = @("https://contoso.sharepoint.com/sites/hr", "https://contoso.sharepoint.com/sites/finance")
+$fontPackage = Get-SPOFontPackage -Identity 12345678-1234-1234-1234-123456789012
+foreach ($site in $sites) {
+    Set-SPOFontPackage -Identity $fontPackage -WebUrl $site
+}
```

Comment:
future improvement: consider updating `WebUrl` param to be a string array, this way you can directly pass all the urls to it, instead of iterating one by one :)